webapp/layouts/default.vue

```diff
@@ -4,7 +4,7 @@
       <ds-container class="main-navigation-container" style="padding: 10px 10px;">
         <div>
           <ds-flex class="main-navigation-flex">
-            <ds-flex-item :width="{ lg: '3.5%' }" />
+            <ds-flex-item :width="{ lg: '30%' }" />
```

Comment:
This looks strange on my machine:
![Screenshot - 2019-08-19T092707 051](https://user-images.githubusercontent.com/2110676/63246610-5aa7ad80-c263-11e9-991b-fab63f56f5cc.png)

Reply:
I forgot to check the view on other devices. Will change it later today.